programmers/K번째수.js

```diff
@@ -0,0 +1,13 @@
+function solution(array, commands) {
+  let answer = [];
+  let sliceSortArr = [];
+  for (let i = 0; i < commands.length; i++) {
+    sliceSortArr = array
+      .slice(commands[i][0] - 1, commands[i][1])
+      .sort((a, b) => {
+        return a - b;
+      });
```

Comment:
```suggestion
      .sort((a, b) => a - b);
```
arrow function에서 하나의 식만 있을 경우 return 생략 가능해요